packages/nc-gui/components/smartsheet/grid/canvas/cells/Lookup.ts

```
@@ -53,6 +54,17 @@ export const LookupCellRenderer: CellRenderer = {
     }
 
     const getArrValue = () => {
+      if (
+        lookupColumn.uidt === UITypes.Checkbox &&
+        [RelationTypes.BELONGS_TO, RelationTypes.ONE_TO_ONE].includes(relatedColObj?.colOptions?.type)
+      ) {
+        const hasLink = !!(row && row[relatedColObj?.title])
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)